pkg/build/builder/docker.go

```diff
@@ -437,6 +461,55 @@ func appendLabel(node *parser.Node, m []dockerfile.KeyValue) error {
 	return appendKeyValueInstruction(dockerfile.Label, node, m)
 }
 
+// appendPostCommit appends a RUN <cmd> Dockerfile instruction as the last child of node
+func appendPostCommit(node *parser.Node, cmd string, uuidAlias string) error {
+	if len(cmd) == 0 {
+		return nil
+	}
+
+	image, alias := getLastFrom(node)
+	if len(alias) == 0 {
+		if len(uuidAlias) != 0 {
+			// so that we can properly test appendPostCommit
```

Comment:
?

why is it only needed for tests?

Reply:
Since a random uuid is created i am unable to check the output to make sure that it's correct (since i won't know what the uuid is), so I pass in a uuid during the tests instead of creating one here.  If there is a better way that you can think of, I will update it.

Reply:
Updated per our IRC conversation and docker alias testing.